src/components/organisms/news/news.scss

```
@@ -0,0 +1,45 @@
+@use "../../variables" as *;
+.news-contents {
+  padding-left: 0px;
+  list-style: none;
+  background-color: $white;
+  margin-bottom: 0;
+  border: solid $gray_gainsboro 1px;
+  border-bottom: 0;
+  .news-content-link {
+    text-decoration: none;
+    .news-content {
+      padding: 19px 30px 19px 20px;
+      border-bottom: solid $gray_gainsboro 1px;
+      transition: ease 0.2s;
+      .news-date {
+        color: $gray_nobel;
+        font-size: $fontsize-xs;
+        line-height: 1;
+        margin-bottom: 10px;
+      }
+      .news-detail {
+        color: $black;
+        font-size: $fontsize-m;
+        line-height: 23px;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        margin-bottom: 0;
+        white-space: nowrap;
+      }
+      &:hover .news-detail {
+        color: $orange_bistre;
+      }
+    }
+  }
+  &:hover .news-content-link {
+    color: $orange_bistre;
+  }
+}
+.information-list {
+  text-align: right;
+  margin-top: 30px;
+  .news-and-events-more {
+    margin-top: 0;
+  }
+}
```

```suggestion
  }
  .information-list {
  text-align: right;
  margin-top: 30px;
  .news-and-events-more {
    margin-top: 0;
    }
  }
}
```
`.information-list` は HTMLの構造上 `.news` の中に入っていると思うので、Scssでもその構造を反映させるようにしましょう。